credentials/academic-credential/credential.json

```
@@ -0,0 +1,68 @@
+{
+    "@context": [
+        "https://www.w3.org/2018/credentials/v1",
+        "https://w3c-ccg.github.io/vc-ed/plugfest-1-2022/jff-vc-edu-plugfest-1-context.json",
+        "https://w3id.org/security/suites/ed25519-2020/v1",
+        "https://purl.imsglobal.org/spec/ob/v3p0/context.json",
```

```suggestion
        "https://purl.imsglobal.org/spec/ob/v3p0/context-3.0.2.json",
```

The `3.0.2` context is the new preferred one for the OpenBadges spec--and by the JFF folks.

---

Good to know! 
I updated and re-validated with the JLD Playground 👍